include/d/actor/d_a_obj_eff.h

```diff
@@ -51,25 +61,36 @@ namespace daObjEff {
         bool _execute();
     
     public:
-        /* Place member variables here */
+        /* 0x290 */ int mProcIndex;
+        /* 0x294 */ dPa_levelEcallBack* mParticleCallback;
```

Comment:
Was there a specific reason this is `dPa_levelEcallBack*`? If not, you could change it to `dPa_followEcallBack*` which would allow you to remove those 12 casts on it in the cpp file.